public/index.html

```diff
@@ -217,16 +218,20 @@
           <section class='col-xs-12 col-sm-6 col-md-6'>
             <section>
               <h2>Deploying code changes</h2>
```

Comment:
it needs to be in general docs for using the templates, we aren't going to doc that in each and every sample app.